src/pages/shopping-cart.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <Page
+    :class="[
+      'w-full',
+      'max-w-[960px]',
+      'mx-auto',
+      'px-[.75em] laptop:px-[2em]',
+      'pb-[8em]',
+      'text-[0.75em] sm:text-[1em]',
+    ]"
+  >
+    <h1
+      :class="[
+        'w-full',
+        'mb-[0.5em]',
+        'text-[#3AB7A2]',
+        'text-[1.5rem] sm:text-[3rem]',
+        'text-left',
+        'font-proxima',
+        'font-[600]',
+      ]"
+    >{{ $t('shopping_cart_title') }}</h1>
+
+    <CardV2
+      v-if="shoppingCartNFTClassList.length"
+      class="w-full laptop:px-[2em]"
+    >
+      <header
+        :class="[
+          'grid',
+          'grid-cols-12',
+          'gap-[1em]',
+          'pl-[1em]',
+          'p-[0.5em]',
+          'border-b-gray-d8',
+          'border-b',
+          'text-gray-9b',
+        ]"
+      >
+        <div class="col-start-1 col-end-8">{{ $t('shopping_cart_list_header_item') }}</div>
+        <!--
+        <div class="col-start-8 col-end-10 text-center">{{ $t('shopping_cart_list_header_price') }}</div>
+        -->
+        <div class="col-start-10 col-end-12 text-center">{{ $t('shopping_cart_list_header_price') }}</div>
+        <!--
+        <div class="col-start-10 col-end-12 text-center">{{ $t('shopping_cart_list_header_quantity') }}</div>
+        -->
+      </header>
+      <ul
+        :class="[
+          'border-b',
+          'border-b-gray-d8',
+        ]"
+      >
+        <li
+          v-for="(item, index) in shoppingCartNFTClassList"
+          :key="item.classId"
+        >
+          <ShoppingCartListRow
+            :index="index + 1"
+            :class-id="item.classId"
+            :quantity="item.quantity"
+            @remove="handleClickRemoveButton"
+          />
+        </li>
+      </ul>
+      <footer class="mt-[1em] text-right">
+        <div class="grid grid-cols-6 items-center gap-[1em]">
+          <div class="col-start-2 sm:col-start-4 sm:col-span-1 text-gray-4a">{{ $t('shopping_cart_list_total_price') }}</div>
+          <div class="col-span-4 sm:col-span-2 text-like-green font-proxima font-[600] leading-1 text-[2em]">{{ totalNFTPrice | formatNumberWithLIKE }}</div>
+          <template v-if="getAddress">
+            <div class="col-start-2 sm:col-start-4 sm:col-span-1 text-gray-4a">{{ $t('shopping_cart_list_wallet_balance') }}</div>
+            <div
+              :class="[
+                'col-span-4 sm:col-span-2 font-proxima font-[600] leading-1',
+                isInsufficientAmount ? 'text-danger' : 'text-like-green',
+              ]"
+            >{{ walletLIKEBalance | formatNumberWithLIKE }}</div>
+          </template>
```

Comment:
Very confusing to have wallet balance under total price, should only show if `isInsufficientAmount`

Reply:
Updated